src/site/content/en/blog/media-session/index.md

```diff
@@ -390,6 +400,79 @@ Resetting the position state is as easy as setting it to `null`.
 navigator.mediaSession.setPositionState(null);
 ```
 
+## Video conferencing actions {: #video-conferencing-actions }
+
+When the user puts their video call into a Picture-in-Picture window, the
+browser may display controls for the microphone, camera, and for hanging up.
+When the user clicks those, the website handles them through the video
+conferencing actions below.
+
+<figure class="w-figure">
+  {% Img
+    src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/fXc7jqc95Oa6sKce7kpZ.jpg",
+    alt="Screenshot of video conferencing controls in a Picture-in-Picture window",
+    width="748",
+    height="464"
+  %}
+  <figcaption class="w-figcaption">Video conferencing controls in a Picture-in-Picture window</figcaption>
+</figure>
+
+{% Aside %}
+At the time of writing, video conferencing actions are available only in
+Chrome&nbsp;92 on desktop.
+{% endAside %}
+
+### Toggle microphone
+
+The `"togglemicrophone"` action indicates that the user wants to mute or unmute
+the microphone. The `setMicrophoneActive(active)` method indicates to the
```

Comment:
It seems like you should replace `active` with something that indicates this is boolean. I don't really care what you call it.

Reply:
Would `isActive` work for you?

Reply:
Done